instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py

```diff
@@ -147,6 +147,19 @@ def test_suppress_instrumentation(self):
 
         self.assert_span(num_spans=0)
 
+    def test_not_recording(self):
+        RequestsInstrumentor().uninstrument()
+        RequestsInstrumentor().instrument(
+            tracer_provider=self.original_tracer_provider
+        )
+
+        result = self.perform_request(self.URL)
+        self.assertEqual(result.text, "Hello!")
+
+        span = self.assert_span(None, 0)
```

Comment:
Looks to me like this would pass regardless of the code change since the behavior isn't actually different.

Would patching `opentelemetry.trace.INVALID_SPAN` and then checking that `is_recording()` was called, `set_attribute()` was never called, etc. work?